src/DDTrace/Integrations/LaravelProvider.php

```diff
@@ -58,7 +58,7 @@ public function register()
             foreach ($pipes as $pipe) {
                 $pipeClass = explode(':', $pipe)[0];
 
-                dd_trace($pipe, 'handle', function (...$args) {
+                dd_trace($pipeClass, 'handle', function (...$args) {
```

Comment:
I may be missing something, but something is not clear to me! Based on #115 this callback receives `....\\Authorize:users`, so a callback. Here looks like that if a pipeline is a callback it is executed (https://github.com/laravel/framework/blob/621d91d802016ab4a64acc5c65f81cb9f5e5f779/src/Illuminate/Pipeline/Pipeline.php#L129).

So I believe that with this proposed approach we don't get the error anymore but we are not tracing the real pipe stop point.

Anyway, very good catch, if you are ok with that, we have already some work ready on a different branch and I will move that forward, does it make sense to you?

Reply:
Hmm didn't look at Pipeline.php. So the problem may be from the `dd_trace` function? 

Sounds good to me 😄

Reply:
It may be! Did you have a chance to investigate at which point the callback `....\\Authorize:users` is sent by Laravel through the pipeline? Just to know how it is invoked, did not have a chance to look at it so far, still managing the various use cases based on the code I see in `Pipeline.php`

Reply:
I know where it comes from but not exactly how it ends up here. It's a middleware so I guess it's logic but don't actually know how. It's used like this:
```
Route::put('/blabla', function (Post $post) {
})->middleware('can:users');
```

Reply:
Ok thanks, looking at your code and properly reading through the Laravel docs now I see the usage you were referring to (I had never seen the middleware with parameters notation before). So I am going to merge your change with another couple of changes that were necessary for similar issues and create a branch. Will you able to build and test that branch?

Reply:
Sounds good! (Though we are not tracing the real call yet right?) 

Yes will be able to build and test 😄